.github/workflows/deploy-enterprise.yml

```diff
@@ -44,9 +44,9 @@ jobs:
             job_files: >-
               job_spec/ARIA_RAIDER.yml
               job_spec/INSAR_ISCE.yml
-            instance_types: c6id.xlarge,c6id.2xlarge,c6id.4xlarge,c6id.8xlarge
-            default_max_vcpus: 0
-            expanded_max_vcpus: 0
+            instance_types: m6id.xlarge,m6id.2xlarge,m6id.4xlarge,m6id.8xlarge,m6idn.xlarge,m6idn.2xlarge,m6idn.4xlarge,m6idn.8xlarge
+            default_max_vcpus: 4000  # Max: 13000
+            expanded_max_vcpus: 4000  # Max: 13000
```

Comment:
@cmarshak we can go up to 13,000 vCPUs in the hyp3-a19-jpl deployment. Do you want to go all the way or stick with 4K for now?